packages/components/src/Table/Table.tsx

```diff
@@ -397,8 +399,9 @@ export function Table<T, U = null>(props: Props<T, U>) {
               entryIndex > 0 && divideBy?.(entries[entryIndex - 1], entryIndex - 1, entries),
           },
           rowClassNames?.className,
-          rowClassNames?.entryRowClassName,
+          getClassName({ className: rowClassNames?.entryRowClassName }, entry, entryIndex, entries),
         )}
+        onClick={() => onCellClick && onCellClick(entry, entryIndex, entries)}
```

Comment:
Can we give this handler more precise name, e.g. onEntryRowClick? Or listen to the clicks on `<td>`?